### PR TITLE
/lyrics Endpoint

### PR DESCRIPTION
**Pull request by PolyAnthi**
Added:
**/lyrics endpoint (uses SRA and LyricsOVH)**
**SQLite Database for caching**

Removed:
**Unwanted Import**

To be removed when merged:
**/coin_value endpoint**

### CODE REVIEW

**Comment by itsnotrin** [comment]
Believe we need to have those qr_code files for the qr code endpoint to work.

**Comment by itsnotrin** [comment]
Also would prefer if we could use an async sqlite3 library, would work better for us in the long term with multiple requests

**Comment by itsnotrin** [comment]
Line 424 on app.py , can you explain why you removed the check for missing lyrics? That'll just cause app errors, no?

**Comment by itsnotrin** [comment]
Denied until these things are fixed

**Review by itsnotrin — CHANGES_REQUESTED** [review]
Listed the changes I'd like done in the conversation.

**Review by PolyAnthi — COMMENTED** [review]
Fixed all stuff Ryan requested and added two new endpoints (truth or dare and Never Have I Ever)

File: app.py [hunk 1]
```diff
@@ -263,16 +297,31 @@ async def joke(simple: Optional[str] = "False"):
...
+    compliment = data[random.randrange(0, len(data))][:-1]
 
     await do_statistics("compliment")
     if simple == "true":
         return PlainTextResponse(compliment)
     return {"success": 1, "data": {"Compliment": compliment}}
 
 
+@app.get("/neverhaveiever")
+async def topic(simple: Optional[str] = "False"):
```
Comment (Dhravya) [inline app.py]: Never have i ever endpoint is named Topic?

File: app.py [hunk 2]
```diff
@@ -247,12 +248,45 @@ async def wyr(simple: Optional[str] = "False"):
...
+        await do_statistics("truthordare")
+        async with aiofiles.open("./data/txt/dare.txt", "r", encoding="utf8") as f:
+            data = await f.readlines()
+        question = data[random.randrange(0, len(data))][:-1]
+
+        if simple == "true":
+            return PlainTextResponse(f"{question}")
+        return {"success": 1, "data": {"Dare": question}}
+    else:
+        return JSONResponse(
```
Comment (Dhravya) [inline app.py]: Any reason to use JSONresponse?

File: app.py [hunk 3]
```diff
@@ -247,12 +248,45 @@ async def wyr(simple: Optional[str] = "False"):
     return {"success": 1, "data": {"Would you rather": question}}
 
 
+@app.get("/truthordare")
+async def truthordare(type: str, simple: Optional[str] = "False"):
```
Comment (Dhravya) [inline app.py]: Better to make a /truth and /dare imo

File: data/txt/truth.txt [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+When's the last time you snored loudly in front of others?
+What's the last big mistake you made in front of others?
+Who's the last person you called by the wrong name?
+Have you ever walked into something while on the phone?
+What did you think at the time was cool but now you regret?
+What did you think you were good at but found out that you really weren't?
+What's your bad breath story?
+What's your snot shooting out my nose story?
+Have you ever clipped your toenails in public?
+What's your really weird secret collection?
```
Comment (Dhravya) [inline data/txt/truth.txt:100]: More truth questions needed

**Review by Dhravya — CHANGES_REQUESTED** [review]
Here are all my reviews, overall great work lol

File: app.py (hunk 1 above)
Comment (PolyAnthi) [reply app.py]: mb LMAO

File: app.py (hunk 2 above)
Comment (PolyAnthi) [reply app.py]: to return 404

File: app.py (hunk 3 above)
Comment (PolyAnthi) [reply app.py]: keep in 1 endpoint

File: data/txt/truth.txt (hunk 4 above)
Comment (PolyAnthi) [reply data/txt/truth.txt:100]: i think 100 for now is good

File: app.py (hunk 3 above)
Comment (Dhravya) [reply app.py]: @itsnotrin Thoughts?

File: app.py (hunk 1 above)
Comment (PolyAnthi) [reply app.py]: fixed

**Comment by PolyAnthi** [comment]
Can we get this merged soon so when we have new code we have a fresh PR?

File: app.py (hunk 3 above)
Comment (Dhravya) [reply app.py]: wait for rin to respond smh

**Review by Dhravya — APPROVED** [review]
everything looks good to me

**Comment by itsnotrin** [comment]
I'll review after my dinner, won't take me long

**Comment by PolyAnthi** [comment]
> I'll review after my dinner, won't take me long

No problem

File: app.py (hunk 3 above)
Comment (itsnotrin) [reply app.py]: Have 1 for both endpoints and 1 for each, so 3 total

**Review by itsnotrin — DISMISSED** [review]
Do what I suggested on wyr please

**Comment by PolyAnthi** [comment]
You mean T or D?

**Comment by itsnotrin** [comment]
I mean make an endpoint for either one and then an endpoint for both
so

/truth
/dare
/truthordare

**Comment by PolyAnthi** [comment]
Ah okay. So truthordare sends both, truth sends a truth and dare sends a
dare

**Comment by itsnotrin** [comment]
Yes

**Comment by PolyAnthi** [comment]
Aight can do that tonight

**Comment by Dhravya** [comment]
Damn this one has been in review since a long time

**Comment by itsnotrin** [comment]
Happy, merging.

**Review by Dhravya — COMMENTED** [review]
looks good

**Pull request merged**